projects/smart-table/src/lib/table-select-filter/table-select-filter.component.html

```diff
@@ -5,7 +5,7 @@
             <div class="a-input__wrapper a-input--filter">
                 <select [id]="id" name="filter-{{ filter.id }}" [(ngModel)]="value" (ngModelChange)="onFilter(value)">
                     <option value="null" disabled="disabled" [selected]="true" *ngIf="filter.placeholder && filter.placeholder.length">{{ filter.placeholder }}</option>
-                    <option *ngFor="let option of filter.options;" [ngValue]="option">{{ option.label }}</option>
+                    <option *ngFor="let option of filter.options;" [attr.value]="option.id">{{ option.label }}</option>
```

Comment:
this is a breaking change, the filter will now send just the id instead of the object (the backing example api in this project happens to be handle both formats, which is why the example still works). I think it's better to send the whole object because it may carry extra properties.